### PR TITLE
enhancement/USERS-Update-users-list

### PR DESCRIPTION
**Pull request by zanusilker**
- Update User list view
- Change Dropdown
- Update router button

### CODE REVIEW

File: src/layouts/app/administrative/users/list/list-item.tsx [hunk 1]
```diff
@@ -12,18 +11,24 @@ import { controller, User, Role } from './controller';
 // components
 import FasIcon from 'components/fas-icon';
 
+import _ from 'services/lodash.service';
+
 // constants
 import * as ROUTES from 'constants/routes';
+import config from '../../../../../configs';
```
Comment (serejabogdan) [inline src/layouts/app/administrative/users/list/list-item.tsx:18]: // configs
import config from 'configs';

File: src/layouts/app/administrative/users/list/list-item.tsx [hunk 2]
```diff
@@ -12,18 +11,24 @@ import { controller, User, Role } from './controller';
...
 
-const ListRow: React.FC<User> = ({ id, name, roles, createdDate }) => {
+const ListItem: React.FC<User> = ({ id, name, roles, createdDate }) => {
```
Comment (serejabogdan) [inline src/layouts/app/administrative/users/list/list-item.tsx:20]: Replace React.FC instead FC

File: src/layouts/app/administrative/users/list/list-item.tsx [hunk 3]
```diff
@@ -48,10 +53,10 @@ const ListRow: React.FC<User> = ({ id, name, roles, createdDate }) => {
         checked={Boolean(selected.length && isChecked)}
       />
```
Comment (serejabogdan) [inline src/layouts/app/administrative/users/list/list-item.tsx:54]: write one line

**Review by serejabogdan — CHANGES_REQUESTED** [review]

File: src/layouts/app/administrative/users/list/actions.tsx [hunk 4]
```diff
@@ -19,55 +19,48 @@ const Actions = () => {
...
   return <Grid container>
-    <Grid xs={12} md={4} item>
+    <Grid item xs={12} md={4}>
       <SearchInput
         value={name}
         disabled={disabled}
         placeholder="Search"
         onInputClear={handleInputClear}
-        onInputApply={handleInputApply}
+        onInputApply={updateFilters}
```
Comment (alexandr-feschenko) [inline src/layouts/app/administrative/users/list/actions.tsx:43]: 🌲

File: src/layouts/app/administrative/users/list/controller.ts [hunk 5]
```diff
@@ -44,9 +44,9 @@ interface Img {
 
 export interface User {
   id: number
-  name: string
-  coverImage: Img
-  createdDate: string
+  name: string | null
+  coverImage: Img |null
+  createdDate: string | null
```
Comment (alexandr-feschenko) [inline src/layouts/app/administrative/users/list/controller.ts:49]: 🌲

**Review by alexandr-feschenko — CHANGES_REQUESTED** [review]

**Pull request merged**